src/components/Alert.tsx

```diff
@@ -1,14 +1,39 @@
 import { useAlert } from '@/context/AlertContext';
+import { clsx } from 'clsx';
+
+export enum AlertColor {
+  Red = 0,
+  Green = 1,
+}
+
+export enum AlertText {
+  SuccessCreatedHistory = 'Successfully created new murojaah',
+  SuccessDeletedHistory = 'Murojaah deleted',
+  FailedCreatedHistory = 'Failed created new murojaah',
+  FailedDeletedHistory = 'Failed deleted murojaah',
+}
 
 export const Alert = (): JSX.Element => {
-  const { isAlertVisible } = useAlert();
+  // @ts-expect-error useAlert
+  const { alertColor, alertText, isAlertVisible } = useAlert();
 
   if (!isAlertVisible) return <></>;
 
+  function getBtnColor(): string {
```

Comment:
this function is executed on every rerender, can but inside useMemo for optimization

Reply:
currently will mark it with some tag for tech debt